sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingAsyncClient.java

```diff
@@ -324,6 +329,117 @@ public FormRecognizerServiceVersion getServiceVersion() {
             continuationToken -> listNextPageModelInfo(continuationToken, context));
     }
 
+    /**
+     * Copy a custom model stored in this resource (the source) to the user specified target Form Recognizer resource.
+     *
+     * <p>This should be called with the source Form Recognizer resource (with the model that is intended to be copied).
+     * The target parameter should be supplied from the target resource's output from
+     * {@link FormTrainingAsyncClient#getCopyAuthorization(String, String)} method.
+     * </p>
+     *
+     * <p>The service does not support cancellation of the long running operation and returns with an
+     * error message indicating absence of cancellation support.</p>
+     *
+     * <p><strong>Code sample</strong></p>
+     * {@codesnippet com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.beginCopyModel#string-copyAuthorization}
+     *
+     * @param modelId Model identifier of the model to copy to the target Form Recognizer resource
+     * @param target the copy authorization to the target Form Recognizer resource
```

Comment:
I would link this to getCopyAuthorization and say it's its output, since users aren't expected to build their own CopyAuthorization class. Python ex [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py#L297)

Reply:
not necessarily. Someone outside of the running app could have shared the copy authorization, so not in all cases we need to call getCopyAuthorization